docs/src/pages/styles/typography/typography-links.example.html

```diff
@@ -0,0 +1,34 @@
+<section class="example--list">
+  <article>
+    <code class="ds-u-margin-y--0 ds-u-margin-bottom--1">.ds-text-heading--5xl</code>
+    <h1 class="ds-text-heading--5xl ds-display"><a href="#">{{lorem-s}}</a></h1>
```

Comment:
This is the only example here that's a link - are these other H tags missing an a tag?

Reply:
Can this be combined with the link.html example that already exists?

Reply:
This is a good call. I was thinking I would go back later to finalize all the documentation but maybe instead I should take out all the documentation from this pull request since it isn't finalized. I'll make a new ticket to finalize typography documentation.

Reply:
Should we be adding documentation to the DSM instead of GitHub per our conversation this morning?